src/common/components/nav/nav.component.js

```diff
@@ -157,7 +157,7 @@ class NavController{
       keyboard: false,
       size: 'sm'
     });
-    this.sessionService.downgradeToGuest().subscribe(() => {
+    this.sessionService[this.customProfile ? 'signOut' : 'downgradeToGuest']().subscribe(() => {
```

Comment:
It's just for the nav component.  Used to for cru.org and staff web.  It won't verify cortex role before displaying name etc.